nodeup/pkg/model/kube_proxy.go

```diff
@@ -150,7 +165,12 @@ func (b *KubeProxyBuilder) buildPod() (*v1.Pod, error) {
 			"/var/log/kube-proxy.log"),
 		Resources: v1.ResourceRequirements{
 			Requests: v1.ResourceList{
-				"cpu": cpuRequest,
+				"cpu":    cpuRequest,
```

Comment:
We'll also need to have conditionals here to avoid adding nil quantities to the ResoureList